openmetadata-ui/src/main/resources/ui/src/pages/AddAlertPage/AddAlertPage.tsx

```diff
@@ -557,8 +559,7 @@ const AddAlertPage = () => {
               <Form.Item
                 label={t('label.description')}
                 labelCol={{ span: 24 }}
-                name="description"
-                rules={[{ required: true }]}>
+                name="description">
```

Comment:
Have you verify it that it's working without filters?

Reply:
@chirag-madlani  yes, it is verified and tested!